services/core-web/src/components/mine/Projects/MajorMineApplicationTab.tsx

```diff
@@ -0,0 +1,191 @@
+import { ScrollSideMenuProps } from "@mds/common/components/common/ScrollSideMenu";
+import { fetchMineDocuments } from "@mds/common/redux/actionCreators/mineActionCreator";
+import { getMineDocuments } from "@mds/common/redux/selectors/mineSelectors";
+import DocumentCompression from "@mds/common/components/documents/DocumentCompression";
+import { Feature, FORM, IProject } from "@mds/common";
+import { MajorMineApplicationDocument } from "@mds/common/models/documents/document";
+import ScrollSidePageWrapper from "@mds/common/components/common/ScrollSidePageWrapper";
+import Loading from "@mds/common/components/common/Loading";
+import { useFeatureFlag } from "@mds/common/providers/featureFlags/useFeatureFlag";
+import ArchivedDocumentsSection from "@mds/common/components/projects/ArchivedDocumentsSection";
+
+
+const MajorMineApplicationTab: FC = () => {
+  const dispatch = useDispatch();
+  const { projectGuid } = useParams<{ tab: string, projectGuid: string }>();
+
+  const majorMineApplication = useSelector(getFormattedProjectApplication);
+  const project: IProject = useSelector(getProject);
+  const majorMineAppStatusCodesHash = useSelector(getMajorMinesApplicationStatusCodesHash);
+  const mineDocuments = useSelector(getMineDocuments);
+  const dropdownMajorMineAppStatusCodes = useSelector(getDropdownMajorMinesApplicationStatusCodes);
+  const archivedDocuments = mineDocuments?.map((doc) => new MajorMineApplicationDocument(doc))
+
+  const defaultLoaded = project.project_guid === projectGuid;
+  const { isFeatureEnabled } = useFeatureFlag();
+  const archiveFeatureEnabled = isFeatureEnabled(Feature.MAJOR_PROJECT_ARCHIVE_FILE);
+  const decisionPackageEnabled = isFeatureEnabled(Feature.MAJOR_PROJECT_DECISION_PACKAGE);
+
+  const [isLoaded, setIsLoaded] = useState(defaultLoaded);
+  const [isCompressionModal, setIsCompressionModal] = useState(false);
+
+  const { major_mine_application_guid, primary_contact, status_code, update_user, update_timestamp, documents = [] } = majorMineApplication;
+
+  const majorMineApplicationDocs = documents.map((doc) => new MajorMineApplicationDocument({
+    ...doc,
+    entity_title: project.project_title
+  }));
+
+  const updateDate = formatDate(update_timestamp);
+
+  const fetchData = async () => {
+    await dispatch(fetchProjectById(projectGuid));
+    if (major_mine_application_guid) {
+      await dispatch(fetchMineDocuments(project.mine_guid, {
+        is_archived: true,
+        major_mine_application_guid
+      }));
+    }
+  };
+
+  useEffect(() => {
+    fetchData().then(() => setIsLoaded(true));
+  }, [projectGuid]);
+
+  const handleUpdateMMA = async (values) => {
+    await dispatch(updateMajorMineApplication({
+      projectGuid,
+      majorMineApplicationGuid: major_mine_application_guid
+    }, values));
+
+    return fetchData();
+  };
+
+  const headerHeight = 121;
+  const tabNavHeight = 60;
+  const topOffset = headerHeight + tabNavHeight;
```

Comment:
at this point, should probably pull this out to the parent component, and/or find a better way. Buuuut it looks and works better now at least.